tracing-opentelemetry-http/build.gradle

```diff
@@ -19,10 +20,11 @@ dependencies {
     compileOnly libs.kotlinx.coroutines.core
     compileOnly libs.kotlinx.coroutines.reactor
 
-    testImplementation mn.reactor
-    testImplementation mn.micronaut.reactor.http.client
+    testImplementation mnReactor.micronaut.reactor
```

Comment:
I think this can remain as `mn.reactor`

Reply:
it has to be `mnReactor.micronaut.reactor` because some code is moved from micronaut-core to micronaut-reactor